etc/kaleidoscope-builder.conf

```diff
@@ -79,7 +79,12 @@ find_device_port() {
     find_device_vid_pid
     DIR=$(dirname "$(readlink -f "$0")")
     DEVICE_PORT_PROBER="${DIR}/find-device-port-linux-udev"
-    DEVICE_PORT="$(perl ${DEVICE_PORT_PROBER} ${VID} ${SKETCH_PID})"
+    if [[ "${DEVICE_PORT}" = "" ]]; then
+      DEVICE_PORT="$(perl ${DEVICE_PORT_PROBER} ${VID} ${SKETCH_PID})"
+    else
+      echo "DEVICE_PORT=\"${DEVICE_PORT}\" predefined."
+      echo "Skipping automatic port search."
+    fi
```

Comment:
Do we want to echo the skip every time? We don't do that for any other thing we can override. Without the echo, this could be simplified to:

```sh
DEVICE_PORT="${DEVICE_PORT:-$(perl ${DEVICE_PORT_PROBER} ${VID} ${SKETCH_PID})}"
```

Reply:
I found a message useful that reports the manually chosen port defined via `DEVICE_PORT`. Is your concern about both messages or just about the "Skipping..."?

Reply:
Both. But I have no strong feelings, and as far as I see (haven't tested, mind you), this should only appear when flashing, which isn't that big a deal.